src/layers/core/grid-layer/grid-layer-vertex.glsl

```diff
@@ -16,25 +15,12 @@ attribute vec3 instancePickingColors;
 uniform float renderPickingBuffer;
 uniform vec3 selectedPickingColor;
 
-// Projection uniforms
-uniform mat4 worldMatrix;
-uniform mat4 viewMatrix;
-uniform mat4 worldInverseTransposeMatrix;
-
 // Custom uniforms
 uniform float extruded;
 uniform float lonOffset;
 uniform float latOffset;
 uniform float opacity;
 
-// Lighting constants
-const vec3 ambientColor = vec3(0.8, 0.8, 0.8);
-const vec3 pointLocation = vec3(1.5, 1.5, 5.);
-const vec3 pointColor = vec3(0.7, 0.7, 0.7);
-const vec3 pointSpecularColor = vec3(0.6, 0.6, 0.6);
-const float shininess = 3.;
```

Comment:
I removed the shininess... I feel that it might be better for us to hardcode it until we have a proper materials system for our users to use